pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +125,61 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+func (pl *VolumeBinding) isSchedulableAfterPersistentVolumeClaimChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldPVC, newPVC, err := util.As[*v1.PersistentVolumeClaim](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"PersistentVolumeClaim", klog.KObj(newPVC),
+	)
+
+	if pod.Namespace != newPVC.Namespace {
+		logger.V(4).Info("PersistentVolumeClaim was created or updated, but it doesn't make this pod schedulable")
+		return framework.QueueSkip, nil
+	}
+
+	for _, vol := range pod.Spec.Volumes {
+		var pvcName string
+		switch {
+		case vol.PersistentVolumeClaim != nil:
+			pvcName = vol.PersistentVolumeClaim.ClaimName
+		case vol.Ephemeral != nil:
+			pvcName = ephemeral.VolumeClaimName(pod, &vol)
+		default:
+			continue
+		}
+
+		if pvcName == newPVC.Name {
+			if oldPVC == nil {
+				logger.V(4).Info("PersistentVolumeClaim was created")
+				return framework.Queue, nil
+			}
+
+			if newPVC.Status.Phase != oldPVC.Status.Phase {
+				logger.V(4).Info("PersistentVolumeClaim referenced by the Pod was created or updated, and changed Phase", "Phase", newPVC.Status.Phase)
+				return framework.Queue, nil
+			}
+
+			if !apiequality.Semantic.DeepEqual(newPVC.Annotations, oldPVC.Annotations) {
+				logger.V(4).Info("PersistentVolumeClaim referenced by the Pod was created or updated, and changed Annotations")
+				return framework.Queue, nil
+			}
+
+			if !apiequality.Semantic.DeepEqual(newPVC.Spec, oldPVC.Spec) {
```

Comment:
L161-L175 would be a large comparison in total, with a little gain.
(esp, DeepEqual is not a lightweight.) 

It feels like we can always return `Queue` simply, like:

```go
// Instead of L156-L175.
if pvcName == newPVC.Name { 
  // Return Queue because, in this case, 
  // all PVC creations and almost all PVC updates could make the Pod schedulable.
  return framework.Queue, nil
}
```